<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Allman recent papers</title>
<style type="text/css">
<!--
body,td,th {
	font-family: Verdana, Geneva, sans-serif;
	font-size: small;
}
-->
</style></head>

<body>
<p><b>Recent Papers and Publications</b>
</p>
  &quot;Split scores: a tool to quantify phylogenetic signal in genome-scale data,&quot<br />
  Elizabeth S. Allman, Laura S. Kubatko and John A. Rhodes,<br />
  Systematic Biology, 
  <strong>66</strong> 4 (2017) 620-636.
<br />

<p>
  SplitSup:  software for computing splits scores.  2016.  Download information on main web page.
<br />

<p>
  &quot;Species tree inference from gene splits by Unrooted STAR methods,&quot;<br />
  Elizabeth S. Allman, James H. Degnan and John A. Rhodes,<br />
  IEEE/ACM Transactions in Computational Biology and Bioinformatics,  <em>to appear</em/>.
<br />

<p>
  &quot;Statistically-Consistent k-mer Methods for Phylogenetic Tree Reconstruction&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, and Seth Sullivant,<br />
  J. of Computational Biology,   
  <strong>24</strong> no. 2 (2017) 153-171. <br/>
  Available on the arXiv or 
  <a href="Papers/kmerSupp.html">here</a> in color.  This link includes the supplementary materials too.
<br />

<p>
  &quot;Parameter identifiability of discrete Bayesian networks with hidden variables&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, Elena Stanghellini, and Marco Valtorta,<br />
  Journal of Causal Inference, 
  <strong>3</strong> no. 2 (2015) 189-205.

<p>
  &quot;Tensors of nonnegative rank two&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, B. Sturmfels, and P. Zwiernik<br />
  Linear Algebra and its Applications, 
  <strong>473</strong> (2015) 37-53.
<br />

<p>
  &quot;A semialgebraic description of the general Markov model on phylogenetic trees&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, and Amelia Taylor<br />
  SIAM J. Discrete Math,
  <strong>28</strong> no. 2 (2014) 736-755.</br>

<p>
  &quot;Tensor Rank, Invariants, Inequalities, and Applications&quot;<br />
  Elizabeth S. Allman, Peter D. Jarvis, John A. Rhodes, and Jeremy G. Sumner<br />
  SIAM Journal on Matrix Analysis and Applications,
  <strong>34</strong> no. 3 (2013) 1014-1045.<br />

<p>
  &quot;Species tree inference by the STAR method, and generalizations&quot;<br />
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes<br />
  J. Computational Biology, <strong>20</strong> no. 1 (2013) 50-61.<br />

<p>
  &quot;When Do Phylogenetic Mixture Models Mimic Other Phylogenetic Models?&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, and Seth Sullivant<br />
  Systematic Biology, <strong>61</strong> no. 6 (2012) 1049-1059.

<p>
  &quot;Determining species tree topologies from clade probabilities under the coalescent&quot;<br />
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes<br />
  Journal of Theoretical Biology, <strong>289</strong> (2011) 96-106.<br />

<p>
  &quot;Parameter Identifiability in a class of random graph mixture models&quot;<br />
  Elizabeth S. Allman, Catherine Matias, and John A. Rhodes<br />
  Journal of Statistical Planning and Inference, <strong>141</strong> no. 5 (2011) 1719-1736. <br/>

<br>
  &quot;Identifying the Rooted Species Tree from the Distribution of Unrooted Gene Trees under the Coalescent&quot;<br />
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes<br />
  Journal of Mathematical Biology, <strong>62</strong> no.6 (2011) 833-862.<br />

<br>
  &quot;Identifiability of 2-tree mixtures for group-based models&quot;<br />
  Elizabeth S. Allman, Sonja Petrovic, John A. Rhodes, and Seth Sullivant<br />
  IEEE/ACM Transactions in Computational Biology and Bioinformatics,
  <strong>8</strong> no. 3 (2011) 710-722. <br />
  Supplementary <a href="http://www4.ncsu.edu/~smsulli2/Pubs/TwoTreesWebsite/twotrees.html">materials</a> <br />

<br>
  &quot;Trees, Fast and Accurate&quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Science</em>,  <strong>327</strong> (2010) 1334-1335. <br/>

<br>
  &quot;Estimating trees from filtered data: Identifiability of models for morphological phylogenetics&quot;<br />
  Elizabeth S. Allman, Mark T. Holder, and John A. Rhodes<br />
  Journal of Theoretical Biology, <strong>263</strong> (2010) 108-119. <br/>

<br>
  &quot;Identifiability of parameters in latent structure models with many observed variables&quot;<br />
  Elizabeth S. Allman, Catherine Matias, and John A. Rhodes<br />
  Annals of Statistics, <strong>37</strong> no.6A (2009) 3099-3132.<br />

<br>
  &quot;The identifiability of <a href="Papers/CovarionIDrev.pdf">Covarion</a> models in phylogenetics&quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  IEEE/ACM Transactions in Computational Biology and Bioinformatics, 
  <strong>6</strong> no. 1 (2009) 76-88.<br />

  <br />
&quot;<a href="Papers/GamId.pdf">Identifiability</a> of a Markovian model of molecular evolution with gamma-distributed rates&quot;<br>
  Elizabeth S. Allman, Cecile Ane, and John A. Rhodes<br />
  <em>Advances in Applied Probability</em>, <strong>40</strong> no. 1 (2008) 228-249.<br />
  Former title was &quot;Identifiability of the GTR+Gamma model of molecular evolution&quot;<br />
  <br>

&quot;The <a href="Papers/rogers.pdf">Gaps</a> in the proof of identifiability of the GTR+Gamma+I model&quot;<br />
  Elizabeth S. Allman, Cecile Ane, and John A. Rhodes<br />
  This is now included as an appendix to the paper listed above.  (June 2007)<br />
  
  <br>
&quot;Identifying evolutionary trees and substitution parameters for
  the general Markov model with invariable sites &quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Mathematical Biosciences</em>, <strong>211</strong> no. 1 (2008) 18-33.<br />
  Appendix materials: Maple and Singular <a href="Papers/gmiCode.txt">code</a>.<br />
  
  <br>
&quot;Phylogenetic Ideals and Varieties for the General Markov
  Model&quot; <br>
  Elizabeth S. Allman and John A. Rhodes&nbsp; <br>
  <span style="font-style: italic;">Advances in Applied Mathematics</span>, <strong>40</strong> no. 2 (2008) 127-148. &nbsp;
  Available (finally) online Jan 10, 2007.<br>
  
  <br>
&quot;Molecular Phylogenetics from an Algebraic Viewpoint
  &quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Statistica Sinica</em>, <strong>17</strong> no. 4 (2007) 1299-1316.<br />
  
  <br />
&quot;Phylogenetic Invariants&quot;<br>
  Elizabeth S. Allman and John A. Rhodes<br>
  Chapter 4 in <span style="font-style:italic;">Reconstructing Evolution New Mathematical and Computational
    Advances</span>,<br>
  edited by O. Gascuel and M. Steel (2007).<br>
  
  <br>
&quot;Phylogenetics&quot;<br>
  Elizabeth S. Allman and John A. Rhodes<br>
  Lecture notes for AMS Short Course &quot;Modeling and
  Simulation of Biological Networks&quot;<br>
  Chapter 2 in <span style="font-style: italic;">Proceedings of Symposia in
    Applied Mathematics</span>, <br />
  edited by R. Laubenbacher (2007).<br> 
 
  <br />
&quot;The identifiability of tree topology for phylogenetic models, including covarion and mixture models&quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Journal of Computational Biology</em>, <strong>13</strong> no. 5 (2006) 1101-1113.<br />

</p>

<a href="papers.html">Other papers</a> are available here.  </p>
</body>
</html>
