<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Research Interests</title>
<!--    <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le styles -->

    <link href="bootstrap-4/css/bootstrap.css" rel="stylesheet">
    <style>
      body {
        margin-top: 0px; /* 60px to make the container go all the way to the bottom of the topbar */
o      }
    </style>
<!--   <meta name="viewport" content="width=2000px">
   <link href="bootstrap-4/css/bootstrap-responsive.css" Rel="stylesheet"> -->
    
   <style>
      myh1 {
          font-size: 35px;
          font-weight: bold;
          color: light-grey;
          letter-spacing: -1px;
          line-height: 2;
          font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
    } 
    </style>

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le fav and touch icons -->
    <link rel="shortcut icon" href="bootstrap/ico/favicon.ico">

    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="bootstrap/ico/apple-touch-icon-114-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="bootstrap/ico/apple-touch-icon-72-precomposed.png">
    <link rel="apple-touch-icon-precomposed" href="bootstrap/ico/apple-touch-icon-57-precomposed.png">

    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="bootstrap/js/bootstrap-transition.js"></script>
    <script src="bootstrap/js/bootstrap-alert.js"></script>
    <script src="bootstrap/js/bootstrap-modal.js"></script>

    <script src="bootstrap/js/bootstrap-dropdown.js"></script>
    <script src="bootstrap/js/bootstrap-scrollspy.js"></script>
    <script src="bootstrap/js/bootstrap-tab.js"></script>
    <script src="bootstrap/js/bootstrap-tooltip.js"></script>
    <script src="bootstrap/js/bootstrap-popover.js"></script>
    <script src="bootstrap/js/bootstrap-button.js"></script>

    <script src="bootstrap/js/bootstrap-collapse.js"></script>
    <script src="bootstrap/js/bootstrap-carousel.js"></script>
    <script src="bootstrap/js/bootstrap-typeahead.js"></script>
</head>

<body>

<div class="container bg-light text-muted">
<!--style="background-color: #17a2b8;">-->

  <div class="row">
    <div class="col-sm-auto">
         <br />
         <img src="Images/fairweather.jpg" height="200" alt="Mt Fairweather" />
         <img src="Images/EA2.jpg" height="200" alt="River to Ridge"  />
    </div>
    <div class="col-sm-auto text-muted">
         <br />
         <br />
          <myh1>Elizabeth S. Allman </myh1><br />
          Mathematics and Statistics • <br />
          UAF, Fairbanks, AK • <br />
          e.allman AT alaska.edu <br />
    </div>
  </div>


<!--<nav class="navbar navbar-light" style="background-color: #e3f2fd;">
      <a class="nav-link active" href="#">Home</a>
</nav>-->

<!--<nav class="navbar navbar-expand-lg navbar-light" style="background-color: #17a2b8;">-->
  <nav class="navbar navbar-expand-lg navbar-light bg-light">
    <div class="collapse navbar-collapse" id="navbarNav">
      <ul class="navbar-nav">
        <li class="nav-item ">
          <a class="nav-link" href="index.html">Home</a>
        <li class="nav-item active">
          <a class="nav-link active " href="research.html">Research</a>
        <li class="nav-item">
          <a class="nav-link" href="teaching.html">Teaching</a>
        <li class="nav-item">
          <a class="nav-link" href="software.html">Software</a>
        <li class="nav-item">
          <a class="nav-link" href="travel.html">Travel</a>
        <li class="nav-item">
          <a class="nav-link" href="photos.html">Photos</a>
    </div>
  </nav>

<!--
  <nav class="nav">
    <a class="nav-link active" href="#">Home</a>
    <a class="nav-link" href="#">Research</a>
    <a class="nav-link" href="#">Teaching</a>
    <a class="nav-link" href="#">Software</a>
    <a class="nav-link" href="#">Travel</a>
    <a class="nav-link" href="#">Photos</a>
   </nav>
-->
</div>

<div class=container>	

<br />

  <p><b>Recent Papers and Publications</b></br></br>

&quot;Parameter identifiability for a profile mixture model of protein evolution.&quot; <br />
S.Yourdkhani, E.S.Allman and J.Rhodes <br />
J. Computational Biology, to appear.

</p>

&quot;MSCquartets 1.0: Quartet methods for species trees and networks under the multispecies
coalescent model in R.&quot; <br />
E.S.Allman, H.D. Banos-Cervantes, J.D. Mitchell and J.Rhodes  <br />
Bioinformatics, to appear. <br />
<a href="https://biorxiv.org/cgi/content/short/2020.05.01.073361v1">bioRxiv</a>, 

  </p>

&quot;Gene tree discord, simplex plots, and statistical tests under the coalescent.&quot; <br />
E.S.Allman, J.D. Mitchell and J.Rhodes  <br />
<a href="https://www.biorxiv.org/content/10.1101/2020.02.13.948083v1">bioRxiv</a>, 
Syst. Biol., to appear. <br />

  </p>

&quot;Testing Multispecies Coalescent Simulators using Summary Statistics,&quot;<br />
 E.S. Allman,  H. Banos-Cervantes and J. Rhodes <br />
 submitted.

  </p>

&quot;NANUQ: A method for inferring species networks
from gene trees under the coalescent model,&quot;   <br />
 E.S. Allman, H. Banos-Cervantes and J.Rhodes   <br />
 Algorithms for Molecular Biology,   <strong>14</strong> no. 24 (2019). <br />
 <a href="https://doi.org/10.1186/s13015-019-0159-2">https://doi.org/10.1186/s13015-019-0159-2</a>
 
  </p>

 &quot;Hypothesis testing near singularities and boundaries,&quot; <br />
 E.S. Allman,  J. Mitchell and J.Rhodes <br />
Electronic Journal of Statistics,
<strong>13</strong> no. 1 (2019) 1250-1293.
  
</p>
  &quot;Species tree inference from genomic sequences using the log-det distance&quot; <br />
  Elizabeth S. Allman, Colby Long, and John A. Rhodes <br />
  SIAM Journal on Applied Algebra and Geometry, 
  <strong>3</strong> no.1 (2019) 107-127. <br />
  
</p>
 &quot;Maximum likelihood estimation of the Latent Class Model through model boundary decomposition&quot;<br />
 with H. Banos-Cervantes, R. Evans, S. Hosten, K. Kubjas, D. Lemke, J. Rhodes, and P. Zwiernik<br />
 Journal of Algebraic Statistics, 
 <strong>10</strong> no. 1 (2019) 3-18. 

</p>
  &quot;Species tree inference from gene splits by Unrooted STAR methods&quot;<br />
  E. Allman, J. Degnan and J. Rhodes <br />
  IEEE/ACM Transactions in Computational Biology and Bioinformatics, 
  <strong>15</strong> no. 1 (2018) 337-342. 

</p>
  &quot;Split probabilities and species tree inference under the multispecies coalescent model&quot;<br />
  E.S. Allman, J.H. Degnan and J.A. Rhodes <br />
  Bulletin of Mathematical Biology,
  <strong>80</strong>, no. 1 (2018) 64-103.

</p>
  &quot;Split scores: a tool to quantify phylogenetic signal in genome-scale data,&quot;<br />
  Elizabeth S. Allman, Laura S. Kubatko and John A. Rhodes,<br />
  Systematic Biology, 
  <strong>66</strong> 4 (2017) 620-636.
<br />

<p>
  SplitSup:  software for computing splits scores.  2016.  Download information on main web page.  Current
  version 1.02
<br />

<p>
  &quot;Statistically-Consistent k-mer Methods for Phylogenetic Tree Reconstruction&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, and Seth Sullivant,<br />
  J. of Computational Biology,   
  <strong>24</strong> no. 2 (2017) 153-171. <br/>
  Available on the arXiv or 
  <a href="papers/kmerSupp.html">here</a> in color.  This link includes the supplementary materials too.
<br />

<p>
  &quot;Parameter identifiability of discrete Bayesian networks with hidden variables&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, Elena Stanghellini, and Marco Valtorta,<br />
  Journal of Causal Inference, 
  <strong>3</strong> no. 2 (2015) 189-205.

<p>
  &quot;Tensors of nonnegative rank two&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, B. Sturmfels, and P. Zwiernik<br />
  Linear Algebra and its Applications, 
  <strong>473</strong> (2015) 37-53.
<br />

<p>
  &quot;A semialgebraic description of the general Markov model on phylogenetic trees&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, and Amelia Taylor<br />
  SIAM J. Discrete Math,
  <strong>28</strong> no. 2 (2014) 736-755.</br>

<p>
  &quot;Tensor Rank, Invariants, Inequalities, and Applications&quot;<br />
  Elizabeth S. Allman, Peter D. Jarvis, John A. Rhodes, and Jeremy G. Sumner<br />
  SIAM Journal on Matrix Analysis and Applications,
  <strong>34</strong> no. 3 (2013) 1014-1045.<br />

<p>
  &quot;Species tree inference by the STAR method, and generalizations&quot;<br />
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes<br />
  J. Computational Biology, <strong>20</strong> no. 1 (2013) 50-61.<br />

<p>
  &quot;When Do Phylogenetic Mixture Models Mimic Other Phylogenetic Models?&quot;<br />
  Elizabeth S. Allman, John A. Rhodes, and Seth Sullivant<br />
  Systematic Biology, <strong>61</strong> no. 6 (2012) 1049-1059.

<p>
  &quot;Determining species tree topologies from clade probabilities under the coalescent&quot;<br />
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes<br />
  Journal of Theoretical Biology, <strong>289</strong> (2011) 96-106.<br />

<p>
  &quot;Parameter Identifiability in a class of random graph mixture models&quot;<br />
  Elizabeth S. Allman, Catherine Matias, and John A. Rhodes<br />
  Journal of Statistical Planning and Inference, <strong>141</strong> no. 5 (2011) 1719-1736. <br/>

<p>
  &quot;Identifying the Rooted Species Tree from the Distribution of Unrooted Gene Trees under the Coalescent&quot;<br />
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes<br />
  Journal of Mathematical Biology, <strong>62</strong> no.6 (2011) 833-862.<br />

<p>
  &quot;Identifiability of 2-tree mixtures for group-based models&quot;<br />
  Elizabeth S. Allman, Sonja Petrovic, John A. Rhodes, and Seth Sullivant<br />
  IEEE/ACM Transactions in Computational Biology and Bioinformatics,
  <strong>8</strong> no. 3 (2011) 710-722. <br />
  Supplementary <a href="http://www4.ncsu.edu/~smsulli2/Pubs/TwoTreesWebsite/twotrees.html">materials</a> <br />

<p>
  &quot;Trees, Fast and Accurate&quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Science</em>,  <strong>327</strong> (2010) 1334-1335. <br/>

<p>
  &quot;Estimating trees from filtered data: Identifiability of models for morphological phylogenetics&quot;<br />
  Elizabeth S. Allman, Mark T. Holder, and John A. Rhodes<br />
  Journal of Theoretical Biology, <strong>263</strong> (2010) 108-119. <br/>

<p>
  &quot;Identifiability of parameters in latent structure models with many observed variables&quot;<br />
  Elizabeth S. Allman, Catherine Matias, and John A. Rhodes<br />
  Annals of Statistics, <strong>37</strong> no.6A (2009) 3099-3132.<br />

<p>
  &quot;The identifiability of Covarion models in phylogenetics&quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  IEEE/ACM Transactions in Computational Biology and Bioinformatics, 
  <strong>6</strong> no. 1 (2009) 76-88.<br />

  <p>
&quot;Identifiability of a Markovian model of molecular evolution with gamma-distributed rates&quot;<br>
  Elizabeth S. Allman, Cecile Ane, and John A. Rhodes<br />
`  <em>Advances in Applied Probability</em>, <strong>40</strong> no. 1 (2008) 228-249.<br />
  Former title was &quot;Identifiability of the GTR+Gamma model of molecular evolution&quot;<br />
  <p>

&quot;The <a href="papers/rogers.pdf">Gaps</a> in the proof of identifiability of the GTR+Gamma+I model&quot;<br />
  Elizabeth S. Allman, Cecile Ane, and John A. Rhodes<br />
  This is now included as an appendix to the paper listed above.  (June 2007)<br />
  
  <p>
&quot;Identifying evolutionary trees and substitution parameters for
  the general Markov model with invariable sites &quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Mathematical Biosciences</em>, <strong>211</strong> no. 1 (2008) 18-33.<br />
  Appendix materials: Maple and Singular <a href="papers/gmiCode.txt">code</a>.<br />
  
  <p>
&quot;Phylogenetic Ideals and Varieties for the General Markov
  Model&quot; <br>
  Elizabeth S. Allman and John A. Rhodes&nbsp; <br>
  <span style="font-style: italic;">Advances in Applied Mathematics</span>, <strong>40</strong> no. 2 (2008) 127-148. &nbsp;
  Available (finally) online Jan 10, 2007.<br>
  
  <p>
&quot;Molecular Phylogenetics from an Algebraic Viewpoint
  &quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Statistica Sinica</em>, <strong>17</strong> no. 4 (2007) 1299-1316.<br />
  
  <p>
&quot;Phylogenetic Invariants&quot;<br>
  Elizabeth S. Allman and John A. Rhodes<br>
  Chapter 4 in <span style="font-style:italic;">Reconstructing Evolution New Mathematical and Computational
    Advances</span>,<br>
  edited by O. Gascuel and M. Steel (2007).<br>
  
  <p>
&quot;Phylogenetics&quot;<br>
  Elizabeth S. Allman and John A. Rhodes<br>
  Lecture notes for AMS Short Course &quot;Modeling and
  Simulation of Biological Networks&quot;<br>
  Chapter 2 in <span style="font-style: italic;">Proceedings of Symposia in
    Applied Mathematics</span>, <br />
  edited by R. Laubenbacher (2007).<br> 
 
  <p>
&quot;The identifiability of tree topology for phylogenetic models, including covarion and mixture models&quot;<br />
  Elizabeth S. Allman and John A. Rhodes<br />
  <em>Journal of Computational Biology</em>, <strong>13</strong> no. 5 (2006) 1101-1113.<br />

  <p>
&quot;Lecture Notes: The Mathematics of Phylogenetics&quot;<br>
  Elizabeth S. Allman and John A. Rhodes<br>
  Park City Mathematics Institute's summer program in Mathematical Biology, undergraduate program<br>
  Preprint, June 26, 2005, 1-127.<br>
  
  <p>
&quot;Phylogenetic invariants for stationary base composition&quot; <br>
  Elizabeth S. Allman and John A. Rhodes<br>
  <em>Journal Symbolic Computation</em>, 41 (2006) 138-150. <br>
  
  <p>
&quot;Quartets and Parameter Recovery for the General Markov Model
  of Sequence Mutation&quot; <br>
  Elizabeth S. Allman and John A. Rhodes <br>
  App. Math. Res. Express</a> Vol. 2004, no. 4, 2004, 107--131.<br>
  
  <p>
&quot;Phylogenetic Invariants for the General Markov Model of
  Sequence Mutation&quot; <br>
  Elizabeth S. Allman and John A. Rhodes 
  Math. Biosci.  186(2), 2003, 113-144, <br>
 <p> </p>

</div>
</body>
</html>
