<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Research Interests</title>
<!--    <meta name="viewport" content="width=device-width, initial-scale=1.0">-->
    <meta name="description" content="">
    <meta name="author" content="">

    <!-- Le styles -->

    <link href="bootstrap-4/css/bootstrap.css" rel="stylesheet">
    <style>
      body {
        margin-top: 0px; /* 60px to make the container go all the way to the bottom of the topbar */
o      }
    </style>
<!--   <meta name="viewport" content="width=2000px">
   <link href="bootstrap-4/css/bootstrap-responsive.css" Rel="stylesheet"> -->
    
   <style>
      myh1 {
          font-size: 35px;
          font-weight: bold;
          color: light-grey;
          letter-spacing: -1px;
          line-height: 2;
          font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
    } 
    </style>

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <!-- Le fav and touch icons -->
    <link rel="shortcut icon" href="bootstrap/ico/favicon.ico">

    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="bootstrap/ico/apple-touch-icon-114-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="72x72" href="bootstrap/ico/apple-touch-icon-72-precomposed.png">
    <link rel="apple-touch-icon-precomposed" href="bootstrap/ico/apple-touch-icon-57-precomposed.png">

    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="bootstrap/js/bootstrap-transition.js"></script>
    <script src="bootstrap/js/bootstrap-alert.js"></script>
    <script src="bootstrap/js/bootstrap-modal.js"></script>

    <script src="bootstrap/js/bootstrap-dropdown.js"></script>
    <script src="bootstrap/js/bootstrap-scrollspy.js"></script>
    <script src="bootstrap/js/bootstrap-tab.js"></script>
    <script src="bootstrap/js/bootstrap-tooltip.js"></script>
    <script src="bootstrap/js/bootstrap-popover.js"></script>
    <script src="bootstrap/js/bootstrap-button.js"></script>

    <script src="bootstrap/js/bootstrap-collapse.js"></script>
    <script src="bootstrap/js/bootstrap-carousel.js"></script>
    <script src="bootstrap/js/bootstrap-typeahead.js"></script>
</head>

<body>

<div class="container bg-light text-muted">
<!--style="background-color: #17a2b8;">-->

  <div class="row">
    <div class="col-sm-auto">
         <br />
         <img src="Images/fairweather.jpg" height="200" alt="Mt Fairweather" />
         <img src="Images/EA2.jpg" height="200" alt="River to Ridge"  />
    </div>
    <div class="col-sm-auto text-muted">
         <br />
         <br />
          <myh1>Elizabeth S. Allman </myh1><br />
          Mathematics and Statistics • <br />
          UAF, Fairbanks, AK • <br />
          e.allman AT alaska.edu <br />
    </div>
  </div>


<!--<nav class="navbar navbar-light" style="background-color: #e3f2fd;">
      <a class="nav-link active" href="#">Home</a>
</nav>-->

<!--<nav class="navbar navbar-expand-lg navbar-light" style="background-color: #17a2b8;">-->
<!--<nav class="navbar navbar-expand-lg navbar-light" style="background-color: #17a2b8;">-->
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <div class="collapse navbar-collapse" id="navbarNav">
      <ul class="navbar-nav">
        <li class="nav-item">
          <a class="nav-link" href="index.html">Home</a>
        <li class="nav-item">
          <a class="nav-link" href="teaching.html">Teaching</a>
        <li class="nav-item">
          <a class="nav-link" href="research.html">Research</a>
        <li class="nav-item">
          <a class="nav-link active" href="Allman-papers.html">Publications</a>
        <li class="nav-item">
          <a class="nav-link" href="software.html">Software</a>
        <li class="nav-item">
	  <a class="nav-link" href="Images/ICERM-2024-advert.pdf">ICERM 2024</a>
        <li class="nav-item">
          <a class="nav-link" href="travel.html">Travel</a>
        <li class="nav-item">
          <a class="nav-link" href="photos.html">Photos</a>
    </div>
  </nav>

</div>

<div class="container text-muted">	

<br />

  <p>Papers and Publications</br>

  <p class="font-weight-bold">2022</p>

  &quot;A generalized AIC for models with singularities and boundaries.&quot;
  with D. Mitchell and J. Rhodes, <em>submitted</em>.
  <p><p>
  
  	&quot;The tree of blobs of a species network:  Identifiability under the coalescent.&quot;
  with H. Banos, D. Mitchell, and  J. Rhodes,
  J. of Mathematical Biology, <em>to appear</em>.

<p><p>
	
	&quot;Parameter identifiability of a multitype pure-birth model of speciation.&quot;
	with D. Dragomir and J. Rhodes, J. of Computational Biology,<em>to apppear</em>.

<p>

&quot;Testing Multispecies Coalescent Simulators using Summary Statistics.&quot;  &nbsp;
 E.S. Allman,  H. Banos-Cervantes and J. Rhodes, 
IEEE/ACM Transactions on Computational Biology and Bioinformatics,
(2022).

<p>	

  &quot;Identifiability of species network topologies from genomic sequences using the logDet distance.&quot;
  E.S. Allman, H. Banos, and J. Rhodes,
  J. Math. Biol., <strong>84</strong> no. 5 (2022) 1–38.
  <a href="https://doi.org/10.1007/s00285-022-01734-2">doi.</a>

<p>
	
&quot;Gene tree discord, simplex plots, and statistical tests under the coalescent.&quot;
E.S. Allman,  J.D. Mitchell and J. Rhodes,
Syst. Biol., <strong>71</strong> no. 4 (2022)  929-942.
  <a href="https://doi.org/10.1007/s00285-022-01734-2"https://doi.org/10.1093/sysbio/syab008">doi.</a>


<p>
  </p>


<p>
  
  <p class="font-weight-bold">2021</p>

</p>
  
&quot;Parameter identifiability for a profile mixture model of protein evolution.&quot;  &nbsp;
S. Yourdkhani, E.S. Allman and J. Rhodes, 
  J. Computational Biology, 
  <strong>28</strong> no. 6 (2021) 570–586. 
<!--<a href="https://www.liebertpub.com/doi/abs/10.1089/cmb.2020.0315">online</a>-->

</p>

&quot;MSCquartets 1.0: Quartet methods for species trees and networks under the multispecies
coalescent model in R.&quot;  &nbsp;
E.S. Allman, H.D. Banos-Cervantes, J.D. Mitchell and J. Rhodes, 
Bioinformatics, <strong>37</strong> 12 (2021) 1766-1768. &nbsp;
 <a href="https://doi.org/10.1093/bioinformatics/btaa868">online</a> </p>
  </p>


<p>

  <p class="font-weight-bold">2019</p>

&quot;NANUQ: A method for inferring species networks
from gene trees under the coalescent model.&quot;    &nbsp;
 E.S. Allman, H. Banos-Cervantes and J.Rhodes,  
 Algorithms for Molecular Biology,   <strong>14</strong> no. 24 (2019). 
 <a href="https://doi.org/10.1186/s13015-019-0159-2">https://doi.org/10.1186/s13015-019-0159-2</a>
 
  </p>

 &quot;Hypothesis testing near singularities and boundaries.&quot; &nbsp;
 E.S. Allman,  J. Mitchell and J.Rhodes, 
Electronic Journal of Statistics,
<strong>13</strong> no. 1 (2019) 1250-1293.
  
</p>
  &quot;Species tree inference from genomic sequences using the log-det distance.&quot; &nbsp;
  Elizabeth S. Allman, Colby Long, and John A. Rhodes, 
  SIAM Journal on Applied Algebra and Geometry, 
  <strong>3</strong> no.1 (2019) 107-127. <br />
  
</p>
 &quot;Maximum likelihood estimation of the Latent Class Model through model boundary decomposition&quot; &nbsp;
 E.S. Allman,  H. Banos-Cervantes, R. Evans, S. Hosten, K. Kubjas, D. Lemke, J. Rhodes, and P. Zwiernik, 
 Journal of Algebraic Statistics, 
 <strong>10</strong> no. 1 (2019) 3-18. 


</p>

   <p class="font-weight-bold">2018</p>

   &quot;Species tree inference from gene splits by Unrooted STAR methods.&quot;
  E. Allman, J. Degnan and J. Rhodes, 
  IEEE/ACM Transactions in Computational Biology and Bioinformatics, 
  <strong>15</strong> no. 1 (2018) 337-342. 

</p>
  &quot;Split probabilities and species tree inference under the multispecies coalescent model.&quot; &nbsp;
  E.S. Allman, J.H. Degnan and J.A. Rhodes, 
  Bulletin of Mathematical Biology,
  <strong>80</strong>, no. 1 (2018) 64-103.

</p>

<p class="font-weight-bold">2017</p>

  &quot;Split scores: a tool to quantify phylogenetic signal in genome-scale data.&quot; &nbsp;
  Elizabeth S. Allman, Laura S. Kubatko and John A. Rhodes,
  Systematic Biology, 
  <strong>66</strong> 4 (2017) 620-636.
<br />

<p><p>
  SplitSup:  software for computing splits scores.  2016.  Download information on main web page.  Current
  version 1.02
<br />

<p>
  
  &quot;Statistically-Consistent k-mer Methods for Phylogenetic Tree Reconstruction.&quot; &nbsp;
  Elizabeth S. Allman, John A. Rhodes, and Seth Sullivant,
  J. of Computational Biology,   
  <strong>24</strong> no. 2 (2017) 153-171. &nbsp;&nbsp;
  Available on the arXiv or 
  <a href="papers/kmerSupp.html">here</a> in color.  This link includes the supplementary materials too.
<br />

<p>

  <p class="font-weight-bold">2015</p>

  &quot;Parameter identifiability of discrete Bayesian networks with hidden variables.&quot; &nbsp;
  Elizabeth S. Allman, John A. Rhodes, Elena Stanghellini, and Marco Valtorta,
  Journal of Causal Inference, 
  <strong>3</strong> no. 2 (2015) 189-205. <br />

<p><p>
  &quot;Tensors of nonnegative rank two.&quot; &nbsp;
  Elizabeth S. Allman, John A. Rhodes, B. Sturmfels, and P. Zwiernik
  Linear Algebra and its Applications, 
  <strong>473</strong> (2015) 37-53.
  <br />


<p>

<p class="font-weight-bold">2014</p>

  &quot;A semialgebraic description of the general Markov model on phylogenetic trees.&quot; &nbsp;
  Elizabeth S. Allman, John A. Rhodes, and Amelia Taylor
  SIAM J. Discrete Math,
  <strong>28</strong> no. 2 (2014) 736-755.</br>

<p><p>

  &quot;A Letter from the Guest Editors,&quot; &nbsp; special issue in 
Mathematical Biology, E.S. Allman, F.Adler and L. de Pillis,
  American Mathematical Monthly, <strong>121</strong> no. 9 (2014) 751-753.<br />

  <p>

<p class="font-weight-bold">2013</p>

  &quot;Tensor Rank, Invariants, Inequalities, and Applications.&quot; &nbsp;
  Elizabeth S. Allman, Peter D. Jarvis, John A. Rhodes, and Jeremy G. Sumner
  SIAM Journal on Matrix Analysis and Applications,
  <strong>34</strong> no. 3 (2013) 1014-1045.<br />

<p><p>
  &quot;Species tree inference by the STAR method, and generalizations.&quot; &nbsp;
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes,
  J. Computational Biology, <strong>20</strong> no. 1 (2013) 50-61.<br />

<p>

<p class="font-weight-bold">2012</p>

  &quot;When Do Phylogenetic Mixture Models Mimic Other Phylogenetic Models?.&quot; &nbsp;
  Elizabeth S. Allman, John A. Rhodes, and Seth Sullivant
  Systematic Biology, <strong>61</strong> no. 6 (2012) 1049-1059.

<p>

<p class="font-weight-bold">2011</p>

  &quot;Determining species tree topologies from clade probabilities under the coalescent.&quot; &nbsp;
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes,
  Journal of Theoretical Biology, <strong>289</strong> (2011) 96-106.<br />

<p><p>
  &quot;Parameter Identifiability in a class of random graph mixture models.&quot; &nbsp;
  Elizabeth S. Allman, Catherine Matias, and John A. Rhodes,
  Journal of Statistical Planning and Inference, <strong>141</strong> no. 5 (2011) 1719-1736. <br/>

<p>
  &quot;Identifying the Rooted Species Tree from the Distribution of Unrooted Gene Trees under the Coalescent.&quot; &nbsp;
  Elizabeth S. Allman, James H. Degnan, and John A. Rhodes,
  Journal of Mathematical Biology, <strong>62</strong> no.6 (2011) 833-862.

<p>
  &quot;Identifiability of 2-tree mixtures for group-based models.&quot; &nbsp;
  Elizabeth S. Allman, Sonja Petrovic, John A. Rhodes, and Seth Sullivant
  IEEE/ACM Transactions in Computational Biology and Bioinformatics,
  <strong>8</strong> no. 3 (2011) 710-722. 
  Supplementary <a href="http://www4.ncsu.edu/~smsulli2/Pubs/TwoTreesWebsite/twotrees.html">materials</a> <br />

<p>

<p class="font-weight-bold">2010</p>

  &quot;Trees, Fast and Accurate.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,
  <em>Science</em>,  <strong>327</strong> (2010) 1334-1335. <br/>

<p><p>
  &quot;Estimating trees from filtered data: Identifiability of models for morphological phylogenetics.&quot; &nbsp;
  Elizabeth S. Allman, Mark T. Holder, and John A. Rhodes,
  Journal of Theoretical Biology, <strong>263</strong> (2010) 108-119. <br/>

<p>

<p class="font-weight-bold">2009</p>

  &quot;Identifiability of parameters in latent structure models with many observed variables.&quot; &nbsp;
  Elizabeth S. Allman, Catherine Matias, and John A. Rhodes,
  Annals of Statistics, <strong>37</strong> no.6A (2009) 3099-3132.<br />

<p><p>
  &quot;The identifiability of Covarion models in phylogenetics.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,
  IEEE/ACM Transactions in Computational Biology and Bioinformatics, 
  <strong>6</strong> no. 1 (2009) 76-88.<br />

<p>

<p class="font-weight-bold">2008</p>

&quot;Identifiability of a Markovian model of molecular evolution with gamma-distributed rates.&quot; &nbsp;
  Elizabeth S. Allman, Cecile Ane, and John A. Rhodes,
  <em>Advances in Applied Probability</em>, <strong>40</strong> no. 1 (2008) 228-249.  &nbsp;&nbsp;
  Former title was &quot;Identifiability of the GTR+Gamma model of molecular evolution.&quot; &nbsp;
  <p>
  <p>
    
&quot;The <a href="papers/rogers.pdf">Gaps</a> in the proof of identifiability of the GTR+Gamma+I model.&quot; &nbsp;
  Elizabeth S. Allman, Cecile Ane, and John A. Rhodes,
  This is now included as an appendix to the paper listed above.  (June 2007)<br />
  
  <p>
&quot;Identifying evolutionary trees and substitution parameters for
  the general Markov model with invariable sites .&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,
  <em>Mathematical Biosciences</em>, <strong>211</strong> no. 1 (2008) 18-33. &nbsp;
  Appendix materials: Maple and Singular <a href="papers/gmiCode.txt">code</a>.<br />
  
  <p>

  <p class="font-weight-bold">2007</p>
  
&quot;Phylogenetic Ideals and Varieties for the General Markov
  Model.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,&nbsp; 
  <span style="font-style: italic;">Advances in Applied Mathematics</span>, <strong>40</strong> no. 2 (2008) 127-148. &nbsp;
  Available (finally) online Jan 10, 2007.<br />
  
  <p><p>
&quot;Molecular Phylogenetics from an Algebraic Viewpoint.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,  &nbsp;
  <em>Statistica Sinica</em>, <strong>17</strong> no. 4 (2007) 1299-1316.<br />
  
  <p>
&quot;Phylogenetic Invariants.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes.  &nbsp;
  Chapter 4 in <span style="font-style:italic;">Reconstructing Evolution New Mathematical and Computational
    Advances</span>, 
  edited by O. Gascuel and M. Steel (2007).<br />
  
  <p>
&quot;Phylogenetics.&quot; &nbsp; 
  Elizabeth S. Allman and John A. Rhodes. &nbsp;
  Lecture notes for AMS Short Course &quot;Modeling and
  Simulation of Biological Networks.&quot; &nbsp;
  Chapter 2 in <span style="font-style: italic;">Proceedings of Symposia in
    Applied Mathematics</span>, 
  edited by R. Laubenbacher (2007).<br /> 
 
  <p>

    
  <p class="font-weight-bold">2006</p>

&quot;The identifiability of tree topology for phylogenetic models, including covarion and mixture models.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes, &nbsp;
  <em>Journal of Computational Biology</em>, <strong>13</strong> no. 5 (2006) 1101-1113.<br />

  <p><p>
&quot;Phylogenetic invariants for stationary base composition.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,
<em>Journal Symbolic Computation</em>, 41 (2006) 138-150. <br />

  <p>
  <p class="font-weight-bold">2005</p>

&quot;Lecture Notes: The Mathematics of Phylogenetics.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes,
  Park City Mathematics Institute's summer program in Mathematical Biology, undergraduate program
  Preprint, June 26, 2005, 1-127.<br />

  <p>
<p class="font-weight-bold">2004</p>

  <p>
&quot;Quartets and Parameter Recovery for the General Markov Model
  of Sequence Mutation.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes, 
  App. Math. Res. Express</a> Vol. 2004, no. 4, 2004, 107-131.<br />
  
<p>

<p class="font-weight-bold">2003</p>

&quot;Phylogenetic Invariants for the General Markov Model of
  Sequence Mutation.&quot; &nbsp;
  Elizabeth S. Allman and John A. Rhodes, 
  Math. Biosci.  186(2), 2003, 113-144, <br />
 <p> </p>

<p class="font-weight-bold">2001</p>

&quot;Division algebras with PSL(2,q)-Galois maximal subfields,&quot; E.S. Allman and 
M. Schacher, J. Algebra, <strong>240</strong> no.2 (2001) 808-821. <br />

<p>
  
<p class="font-weight-bold">1999</p>

&quot;Abelian subgroups of finitely generated Kleinian groups are
separable.&quot; E.S. Allman and E. Hamilton, Bulletin of the London
Mathematical Society, </strong>31</strong> no. 2 (1999) 163-172.

<p>
  
<p class="font-weight-bold">1996</p>

&nbsp;Polynomials Without Roots in Division Algebras.&quot &nbsp;
Communications in Algebra, </strong>24</strong> no. 13 (1996) 3891-3919.

 
</div>
</body>
</html>
